### PR TITLE
Add DiffEq path animation

### PR DESCRIPTION
**Pull request by SebastianM-C**
Add an example on how to animate a solution from DifferentialEquations.jl (@ChrisRackauckas is this ok?).

### CODE REVIEW

File: examples/examples3d.jl [hunk 1]
```diff
@@ -710,3 +710,41 @@ end
...
+        xm, xM = extrema(sol[1,:])
+        ym, yM = extrema(sol[2,:])
+        zm, zM = extrema(sol[3,:])
+
+        limits = FRect3D((xm,ym,zm), (xM-xm,yM-ym,zM-zm))
+
+        scene = lines(trajectory, limits=limits, markersize=0.7)
+        meshscatter!(scene, endpoint, limits=limits, markersize=0.5)
+
+        ts = range(tspan..., step=0.01)
```
Comment (asinghvi17) [inline examples/examples3d.jl]: Should this be shorter?  Maybe:

```julia
ts = LinRange(tspan..., 90)
record(scene, @replace_with_a_path(mp4), ts, framerate = 30) do tᵢ
...
```

Comment (asinghvi17) [reply examples/examples3d.jl]: This is currently a 2 minute video :P

Comment (SebastianM-C) [reply examples/examples3d.jl]: I shortened the timespan 3 times, so it should be about 40s. Is it ok or should I shorten it more?

File: examples/examples3d.jl [hunk 2]
```diff
@@ -710,3 +710,41 @@ end
...
+        using OrdinaryDiffEq
+
+        function lorenz(du,u,p,t)
+            du[1] = 10.0*(u[2]-u[1])
+            du[2] = u[1]*(28.0-u[3]) - u[2]
+            du[3] = u[1]*u[2] - (8/3)*u[3]
+        end
+
+        u0 = [1.0; 0.0; 0.0]
+        tspan = (0.0, 10.0)
```
Comment (SebastianM-C) [inline examples/examples3d.jl]: It simpler to just shorten the timespan. Changing the number of time points in the animation (`ts`) can lead to a poor sampling of the trajectory, due to large jumps.

File: examples/examples3d.jl (hunk 1 above)
Comment (asinghvi17) [reply examples/examples3d.jl]: 40s should be reasonable imo.

**Comment by asinghvi17** [comment]
Pinging @SimonDanisch for final review.

**Comment by SimonDanisch** [comment]
I changed the camera a bit:
test
Does that look fine to you (it does look much better with higher N, but then the gif becomes too large to upload it here^^)?

**Comment by SebastianM-C** [comment]
It looks a bit strange / jumpy with the camera fixed on the endpoint of the trajectory, but I guess that it's also an example of how to do that :)))

**Comment by SimonDanisch** [comment]
Yeah, it does looks much smoother with more points, though (but still a bit weird admitattely, but just going roughly into the direction of the endpoint would be much more complicated) ;)

**Comment by asinghvi17** [comment]
@SimonDanisch could you push your changes to this branch as well?  Thanks!

**Comment by SimonDanisch** [comment]
Thank you! I merged an improved version in: https://github.com/JuliaPlots/MakieGallery.jl/pull/48

**Pull request closed without merging**